keycloak_client.go

```diff
@@ -419,6 +408,24 @@ func str(s string) *string {
 	return &s
 }
 
+func whitelistErrors(statusCode int, message string) error {
+	// whitelist errors from Keycloak
+
+	switch message {
+	//POST account/credentials/password with error message "invalidPasswordExistingMessage"
+	case "invalidPasswordExistingMessage":
```

Comment:
May be we should have a map/array of all whitelisted messages (hard-coded or loaded from an input file) and just have something like :
```
if _, ok:=whitelisted[message], ok {
    return commonhttp.Error{...}
} else {
    return HTTPError{...}
}
```